storage/client_test.go

```diff
@@ -215,6 +215,19 @@ func incrementArgs(key []byte, inc int64, raftID int64, storeID proto.StoreID) (
 	return args, reply
 }
 
+func internalTruncateLogArgs(index uint64, raftID int64, storeID proto.StoreID) (
```

Comment:
we already have this in `storage/range_test.go`, I assume there wasn't a good way to be DRY.